cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverPluginTest.java

```diff
@@ -242,18 +245,9 @@ private void testLogRead(LoggingContext loggingContext) throws Exception {
     final Multimap<String, String> contextMessages = getPublishedKafkaMessages();
```

Comment:
Since we are printing all messages in the outer catch block, we don't need to print it here, right?